music.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
+
+const developerKey = ""
```

Comment:
Change this so that this variable is declared in `main.go` and the key is read from the `.env` file as `YOUTUBE_API_KEY`

Reply:
Working on it.

Reply:
Done. Found some other issues in music.go and fixed them as well.